frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -20,10 +20,21 @@ export const TaskOverviewPage = () => {
   }>();
   const history = useHistory();
   const task = useSelector(taskSelector(Number(taskId)));
+  const tasks = useSelector(allTasksSelector(), shallowEqual);
+  const taskIdx = tasks.findIndex(({ id }) => Number(taskId) === id);
```

Comment:
```suggestion
  const tasks = useSelector(allTasksSelector(), shallowEqual);
  const taskIdx = tasks.findIndex(({ id }) => Number(taskId) === id);
  const task = taskIdx >= 0 ? tasks[taskIdx] : undefined;
```
The current task could *maybe* be taken from the all tasks list